### PR TITLE
Update documentation about database

### PR DESCRIPTION
**Pull request by guillaumelamirand**
Thank you for your contribution to the Axway Streams documentation.

## Describe the changes

Enter a brief description of your changes here to communicate to the maintainers what you changed and why.

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

## Checklist for approvers

_This section is to be filled out by project maintainers only._

Before approving this PR, please make sure it:

* [ ] Does not have conflicts with the base branch
* [ ] Is believed to be accurate (technical accuracy to be checked with an SME for PRs originating from outside R&D)
* [ ] Follows [Axway style](https://techweb.axway.com/confluence/display/RIE/Style+guide)
* [ ] Follows [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)
* [ ] Follows [best practices](https://axway-open-docs.netlify.com/docs/contribution_guidelines/bestpracticedevdoc/)
* [ ] Does not contain typos, grammatical errors, etc.
* [ ] Does not expose any sensitive information
* [ ] Passes the Markdown linter rules (automated via CI check)
* [ ] Does not contain broken links

### CODE REVIEW

File: content/en/docs/streams/install/_index.md [hunk 1]
```diff
@@ -41,78 +43,77 @@ We recommend deploying Streams components inside a dedicated namespace. To creat
...
-You must create a secret storing those passwords doing:
+You can follow the officiel documentation provided by Mariadb [Certificate Creation with OpenSSL](https://mariadb.com/kb/en/certificate-creation-with-openssl/) to generate self-signed certificate. *Don't forget the set the right Common Name.*
 
-```bash
-kubectl create secret generic streams-crypto-password-secret --from-literal=hub=<hub crypto password> --from-literal=subscriberWebhook=<webhook crypto password> --from-literal=subscriberKafka=<kafka crypto password> -n my-namespace
-```
+### Transparent Data Encryption
 
-The three literals in the secret are used by their respective microservice to encrypt their data.
+Mariadb data-at-rest encryption is also enabled by default, so you must provide a keyfile .
```
Comment (jfmarquet) [inline content/en/docs/streams/install/_index.md:77]: remove final space after keyfile

File: content/en/docs/streams/install/_index.md [hunk 2]
```diff
@@ -41,78 +43,77 @@ We recommend deploying Streams components inside a dedicated namespace. To creat
...
+The keyfile must contain a 32-bit integer identifier following by the hex-encoded encryption key separated by semicolon such as : <encryption_key_id>;<hex-encoded_encryption_key>.
```
Comment (jfmarquet) [inline content/en/docs/streams/install/_index.md:78]: followed by
remove space after "such as"

File: content/en/docs/streams/install/_index.md [hunk 3]
```diff
@@ -41,78 +43,77 @@ We recommend deploying Streams components inside a dedicated namespace. To creat
...
 
 ```sh
-export POSTGRES_ADMIN_PASSWORD="YourPostgresAdminPassword"
-export POSTGRES_USER_PASSWORD="YourPostgresUserPassword"
-export POSTGRES_REPLICATION_PASSWORD="YourPostgresReplicationPassword"
-kubectl create secret generic streams-postgresql-password-secret --from-literal=postgresql-password=${POSTGRES_ADMIN_PASSWORD} --from-literal=postgresql-postgres-password=${POSTGRES_ADMIN_PASSWORD} --from-literal=postgresql-streams-password=${POSTGRES_USER_PASSWORD} --from-literal=postgresql-replication-password=${POSTGRES_REPLICATION_PASSWORD} -n my-namespace
+export MARIADB_ROOT_PASSWORD="YourMariadbRootPassword"
+export MARIADB_PASSWORD="YourMariadbUserPassword"
+export MARIADB_REPLICATION_PASSWORD="YourMariadReplicationPassword"
+kubectl create secret genericstreams-database-password-secret --from-literal=mariadb-root-password=${MARIADB_ROOT_PASSWORD} --from-literal=mariadb-password=${MARIADB_PASSWORD}  --from-literal=mariadb-replication-password=${MARIADB_REPLICATION_PASSWORD} -n my-namespace
```
Comment (jfmarquet) [inline content/en/docs/streams/install/_index.md:64]: add space between generic and streams-database-password-secret

File: content/en/docs/streams/architecture/_index.md [hunk 4]
```diff
@@ -679,28 +679,28 @@ Pod characteristics for HA deployment mode:
...
-* Repository: bitnami/postgresql
-* Tag: 12.4.0
+* Repository: bitnami/mariadb
+* Tag: 10.4.14
 
 Pod names:
 
-* `streams-postgresql-master`
-* `streams-postgresql-slave`
+* `streams-mariadb-master`
```
Comment (jfmarquet) [inline content/en/docs/streams/architecture/_index.md:693]: streams-database-master-0

File: content/en/docs/streams/architecture/_index.md [hunk 5]
```diff
@@ -679,28 +679,28 @@ Pod characteristics for HA deployment mode:
...
+* `streams-mariadb-slave`
```
Comment (jfmarquet) [inline content/en/docs/streams/architecture/_index.md:694]: streams-database-slave-0

File: content/en/docs/streams/install/_index.md [hunk 6]
```diff
@@ -41,78 +43,77 @@ We recommend deploying Streams components inside a dedicated namespace. To creat
...
+```
 
-SSL/TLS is *enabled by default* on our embedded Ingress controller. You need to provide an SSL/TLS certificate for the domain name you are using:
+### Secrets
+
+Depending on your security choices, you must create a secret containing both TLS certificates and TDE keyfile or either one of them: 
 
 ```sh
-kubectl create secret tls streams-ingress-tls-secret --key=<path/to.key> --cert=<path/to-full-chain.crt> -n my-namespace
+kubectl create secret generic streams-database-secret --from-literal=CA_PEM="$(cat ca.pem)" --from-literal=SERVER_CERT_PEM="$(cat server-cert.pem)" --from-literal=SERVER_KEY_PEM="$(cat server-key.pem)" --from-literal=KEYFILE="$(cat keyfile)" -n my-namespace
```
Comment (jfmarquet) [inline content/en/docs/streams/install/_index.md:91]: export my-namespace before command

File: content/en/docs/streams/install/_index.md [hunk 7]
```diff
@@ -41,78 +43,77 @@ We recommend deploying Streams components inside a dedicated namespace. To creat
...
 ```
 
-### PostgreSQL TLS settings
+or only for TLS
 
-By default, Streams Helm will set up TLS communication between PostgreSQL and Streams microservices, so you must provide a certificate and its key. Below the required steps using a self-signed certificate:
+```sh
+kubectl create secret generic streams-database-secret --from-literal=CA_PEM="$(cat ca.pem)" --from-literal=SERVER_CERT_PEM="$(cat server-cert.pem)" --from-literal=SERVER_KEY_PEM="$(cat server-key.pem)" -n my-namespace
```
Comment (jfmarquet) [inline content/en/docs/streams/install/_index.md:97]: export my-namespace before command

File: content/en/docs/streams/install/_index.md [hunk 8]
```diff
@@ -41,78 +43,77 @@ We recommend deploying Streams components inside a dedicated namespace. To creat
...
+```
 
-* To create a self-signed certificate, use the following OpenSSL command: (only for testing purposes):
+or only for TDE
 
 ```sh
-openssl req -x509 -newkey rsa:2048 -keyout server.key -out server.crt -text -nodes -subj '/CN=streams-postgresql'
+kubectl create secret generic streams-database-secret --from-literal=KEYFILE="$(cat keyfile)" -n my-namespace
```
Comment (jfmarquet) [inline content/en/docs/streams/install/_index.md:103]: export my-namespace before command

File: content/en/docs/streams/install/_index.md [hunk 9]
```diff
@@ -41,78 +43,77 @@ We recommend deploying Streams components inside a dedicated namespace. To creat
...
 
-Make sure to set the postgresql service name `streams-postgresql` as `Common Name`; All other should be left. blank using `.` as answer.
+### Ingress TLS settings
 
-Then you must create a secret storing those files doing:
+SSL/TLS is *enabled by default* on our embedded Ingress controller. You need to provide an SSL/TLS certificate for the domain name you are using:
 
 ```sh
-kubectl create secret generic postgresql-certificates-secret --from-file=./server.crt --from-file=./server.key -n my-namespace
+kubectl create secret tls streams-ingress-tls-secret --key=<path/to.key> --cert=<path/to-full-chain.crt> -n my-namespace
```
Comment (jfmarquet) [inline content/en/docs/streams/install/_index.md:111]: export my-namespace before command

File: content/en/docs/streams/install/_index.md [hunk 10]
```diff
@@ -41,78 +43,77 @@ We recommend deploying Streams components inside a dedicated namespace. To creat
...
 ```
 
 ### Helm command
 
-The command below deploys Streams on the Kubernetes cluster in High availability mode. The passwords for postgreSQL database and Hazelcast must be defined in the installation command line. By doing so, they are stored securely inside the k8s cluster and not visible in plaintext in `values.yaml` file. There are optional parameters that can be specified to customize the installation.
+The command below deploys Streams on the Kubernetes cluster in High availability mode. By doing so, they are stored securely inside the k8s cluster and not visible in plaintext in `values.yaml` file. There are optional parameters that can be specified to customize the installation.
```
Comment (jfmarquet) [inline content/en/docs/streams/install/_index.md:116]: removing the passwords sentence invalidates the following one "by doing so...". I would remove the sentence "By doing so..."

File: content/en/docs/streams/install/_index.md [hunk 11]
```diff
@@ -22,6 +22,8 @@ cd <YOUR_INSTALL_DIR>/helm/streams
 
 ## Helm Chart installation
 
+You can refere to Kubernetes documentation to create [secrets](https://kubernetes.io/docs/concepts/configuration/secret/)
```
[the remark this answers is not in the document]
Comment (guillaumelamirand) [reply content/en/docs/streams/install/_index.md:25]: I didn't know what to do with that as we don't have a secret section anymore. I will add a section only with the link

**Review by allandenis38 — APPROVED** [review]

**Pull request merged**